infra/terraform/global/main.tf

```diff
@@ -65,3 +65,41 @@ module "hmpps_oasys_upload_user" {
   org_name          = "hmpps"
   system_name       = "oasys"
 }
+
+// Backup etcd volumes attached to kubernetes masters -->
```

Comment:
Is the mix of `#` and `//` style comments significant?

Reply:
No